src/qililab/qprogram/experiment_executor.py

```diff
@@ -112,6 +114,10 @@ def __init__(
         self,
         platform: "Platform",
         experiment: Experiment,
+        database: bool = True,
+        db_manager: DatabaseManager | None = None,
+        base_path: str | None = None,
+        optional_identifier: str | None = None,
```

Comment:
Why do we need all these information here since we have access to the `platform`?

Platform is the one responsible for handling the database management, so we shouldn't have duplicated source of truths.

Users will `platform.load_db_manager()` and give all information needed once there.